docs/photoniq/vwrs/configure-vwrs-edgeworker.md

```diff
@@ -0,0 +1,170 @@
+- **statusConfigLimits**: This defines the waiting room information that is returned by the status call. Use the following three values to define what the status page returns:
+  - **avgWaitingTime**: If set to `true`, then the average waiting time is returned by the status call. (default: `true`)
+  - **qDepth**: If set to `true`, then the waiting room queue size is returned. (default: `true`)
+  - **position**: If set to `true`, then the position of the request in the waiting room queue is returned. (default: `true`)
+
+## Request Configuration
+
+A request configuration is an optional object that contains configuration parameters for the request handler. Options include:
+
+- **waitingRoomPath**: The cloud origin (such as NetStorage) path to the waiting room HTML.
+- **extraFingerprint**: Additional data to include in the fingerprint calculation.
+- **debugMode**: This flag enables debug mode. In debug mode, the unencrypted cookie is stored in the `x-vwrs-debug` header and logged to the logger (default: `false`).
+- **priority:** A positive whole number indicates the priority of the request when a priority queue is enabled for the specific waiting room. This priority number must be one of the priorities configured for the waiting room.  
+
+## Fingerprinting
+
+VWRs can use device fingerprinting techniques to help mitigate cookie sharing and replay attacks. By default, a basic fingerprint is generated using information derived from the device object ([https://techdocs.akamai.com/edgeworkers/docs/device-object](https://techdocs.akamai.com/edgeworkers/docs/device-object)). If stronger device fingerprinting is required, then you can override this logic with your own, or configure it to use one of the fingerprinting techniques available in Akamai’s security products.
+
+## Priority
+
+You can customize a waiting room with up to ten priorities, the priorities being positive numbers from 1 through 255. Configure the priorities using the `POST/PUT/PATCH /api/vwr/v1/domains` REST API.
+
+For instance, consider configuring three priorities with values of 1, 20, and 255. To accomplish this, you can use the `PATCH /api/vwr/v1/domains` REST API to update the domain:
+
+```bash
+curl -X 'PATCH' \
+   https://api-vwr-service.gdn-akamai.com/api/vwr/v1/origins/example3.com \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'authorization: test_key.12345678abcdefghijklABCDEFGHIJKL' \
+  -d '{
+  "priority": [1, 20, 255]
+}'
+```
+
+You have now successfully configured the waiting room to acknowledge priorities. To prioritize the traffic, you must pass the priority to the `handleVwrsRequest()` and `handleVwrsResponse()` functions. Here's a template demonstrating how to include a priority with the request in the `handleVwrsRequest()` call:
```

Comment:
"...to acknowledge priorities."  @rahul-naik1, what does "acknowledge priorities." mean?
Do we mean 
"You have now successfully configured the waiting room to support priorities."

Reply:
I agree with @joubert-macrometa Let's update it.